apps/api/src/config/index.ts

```diff
@@ -4,26 +4,15 @@ import { str, url, port, ValidatorSpec } from 'envalid';
 
 dotenv.config();
 
-let path;
-switch (process.env.NODE_ENV) {
-  case 'prod':
-    path = `${__dirname}/../.env.production`;
-    break;
-  case 'test':
-    path = `${__dirname}/../.env.test`;
-    break;
-  case 'ci':
-    path = `${__dirname}/../.env.ci`;
-    break;
-  case 'local':
-    path = `${__dirname}/../.env.local`;
-    break;
-  case 'dev':
-    path = `${__dirname}/../.env.development`;
-    break;
-  default:
-    path = `${__dirname}/../.env.local`;
-}
-//
+const path =
+  `${__dirname}/../` +
+  ({
+    prod: '.env.production',
+    test: '.env.test',
+    ci: '.env.ci',
+    local: '.env.local',
+    dev: '.env.development',
+  }[process.env.NODE_ENV] || '.env.local');
```

Comment:
I really like that, much better than the repeated switch casing! 

Can we just move the object map to a different `const` so it will be easier for the reader to understand what happens here (Following the [Extract Variable](https://refactoring.guru/extract-variable) refactoring pattern)? 
```typescript
const envFilesMap = {
    prod: '.env.production',
    test: '.env.test',
    ci: '.env.ci',
    local: '.env.local',
    dev: '.env.development',
};
```

And we will have the `path`:
```typescript
const selectedEnvFile = (envFilesMap[process.env.NODE_ENV] || '.env.local');
const path = `${__dirname}/../${selectedEnvFile}`;
```

Feel free to find better names for the new variables, or their responsibility :)

Reply:
Yes, I agree.
The Extract Variable method helps to have more readable code.